### PR TITLE
Greatly shorten the Getting data into and out of an event listener section

### LINKED ISSUE #23253: Clarifying scope in event listener definition and execution
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#getting_data_into_an_event_listener_using_the_outer_scope_property

### What specific section or headline is this issue about?

In the code example and the "Note" below it.

### What information was incorrect, unhelpful, or incomplete?

The information about defining the event listener seemed confusing.

### What did you expect to see?

Something similar to the difference between defining a function versus calling one.

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### PR DESCRIPTION
**Pull request by Josh-Cena**
Fix https://github.com/mdn/content/issues/23253

Maybe it was an issue when there was no `const`/`let`, but it no longer is. All this text just seems like bloat.

### CODE REVIEW

File: files/en-us/web/api/eventtarget/addeventlistener/index.md [hunk 1]
```diff
@@ -345,119 +345,26 @@ myObject.register();
 
 ### Getting data into and out of an event listener
 
-It may seem that event listeners are like islands, and that it is extremely difficult
-to pass them any data, much less to get any data back from them after they execute.
 Event listeners only take one argument,
 the [Event Object](/en-US/docs/Learn/JavaScript/Building_blocks/Events#event_objects),
```
Comment (wbamberg) [inline files/en-us/web/api/eventtarget/addeventlistener/index.md:349]:
```suggestion
an {{domxref("Event")}} or a subclass of `Event`,
```

File: files/en-us/web/api/eventtarget/addeventlistener/index.md [hunk 2]
```diff
@@ -345,119 +345,26 @@ myObject.register();
...
-#### Getting data into an event listener using the outer scope property
-
-When an outer scope contains a variable declaration (with `const`,
-`let`), all the inner functions declared in that scope have access to that
-variable (look [here](/en-US/docs/Glossary/Function#different_types_of_functions) for
-information on outer/inner functions, and [here](/en-US/docs/Web/JavaScript/Reference/Statements/var#implicit_globals_and_outer_function_scope)
-for information on variable scope). Therefore, one of the simplest ways to access data
-from outside of an event listener is to make it accessible to the scope in which the
-event listener is declared.
+The functions passed as event listeners have access to all variables declared in the outer scopes that contain the function. Read [the function guide](/en-US/docs/Web/JavaScript/Guide/Functions#function_scope) for more information about function scopes. Therefore, you can access any data from outside of an event listener is to make it accessible to the scope in which the event listener is declared.
```
Comment (wbamberg) [inline files/en-us/web/api/eventtarget/addeventlistener/index.md:353]: The sentence 3 doesn't make sense to me, even grammatically.

I think maybe you don't even need it, given sentence 1.

And I would personally move sentence 2 after the example code, to make the connection between sentence 1 and the example stronger, and sentence 2 more of a "see also".

**Review by wbamberg — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/eventtarget/addeventlistener/index.md (hunk 2 above)
Comment (Josh-Cena) [reply files/en-us/web/api/eventtarget/addeventlistener/index.md:353]: Oh my, what a terrible sentence it is. I just didn't delete enough words.

File: files/en-us/web/api/eventtarget/addeventlistener/index.md [hunk 3]
```diff
@@ -345,118 +345,27 @@ myObject.register();
...
-event listener is declared.
+The functions passed as event listeners have access to all variables declared in the outer scopes that contain the function.
 
 ```js
 const myButton = document.getElementById("my-button-id");
 let someString = "Data";
 
 myButton.addEventListener("click", () => {
-  console.log(someString); // Expected Value: 'Data'
+  console.log(someString); // Expected Value: 'Data' on first click, 'Data Again' on second click
```
Comment (wbamberg) [inline files/en-us/web/api/eventtarget/addeventlistener/index.md:360]: Sorry to be picky, and not to have noticed this first time.

This is >80 characters so the comment line gets hard-wrapped on MDN. Could consider either:
- putting it on its own line
- removing "Expected Value" here and in the other line (would still need to find a couple of characters form somewhere else)

Comment (Josh-Cena) [reply files/en-us/web/api/eventtarget/addeventlistener/index.md:360]:
```suggestion
  console.log(someString);
  // 'Data' on first click,
  // 'Data Again' on second click
```

**Review by wbamberg — APPROVED** [review]
👍 nice, thank you @Josh-Cena !

**Pull request merged**